mypy/main.py

```diff
@@ -451,6 +451,7 @@ def get_init_file(dir: str) -> Optional[str]:
     'python_version': lambda s: tuple(map(int, s.split('.'))),
     'strict_optional_whitelist': lambda s: s.split(),
     'custom_typing_module': str,
+    'mypy_path': lambda s: [p.strip() for p in s.split(',')],
```

Comment:
I think it's better to stick to [a common convention](https://github.com/search?utf8=%E2%9C%93&q=exclude+node_modules+extension%3A.cfg&type=Code&ref=searchresults) of handling lists in config files. Parsing that setting this way would probably cause confusion.